pkg/ucp/frontend/api/routes.go

```diff
@@ -131,64 +125,17 @@ func Register(ctx context.Context, router chi.Router, planeModules []modules.Ini
 
 	// Configures planes collection and resource routes.
 	planeCollectionRouter := server.NewSubrouter(router, options.PathBase+planeCollectionPath, apiValidator)
-	planeTypeRouter := server.NewSubrouter(router, options.PathBase+planeCollectionByTypePath, apiValidator)
-	planeNameRouter := server.NewSubrouter(router, options.PathBase+planeCollectionByTypePath+"/{planeName}", apiValidator)
 
+	// The "list all planes by type" handler is registered here.
 	handlerOptions = append(handlerOptions, []server.HandlerOptions{
 		// Planes resource handler registration.
 		{
-			// This is scope query unlike the default list handler.
+			// This is a custom controller because we have to use custom query logic to list planes of all types.
 			ParentRouter:      planeCollectionRouter,
 			Method:            v1.OperationList,
 			OperationType:     &v1.OperationType{Type: OperationTypePlanes, Method: v1.OperationList},
 			ControllerFactory: planes_ctrl.NewListPlanes,
 		},
-		{
```

Comment:
Now that each plane has it's own converters, the route registration for the plane APIS needed to be plane-specific. This is now handled by the individual modules for each plane type.